txstratum/manager.py

```diff
@@ -136,6 +137,7 @@ async def update_block_template(self) -> None:
         except Exception:
             # XXX What should we do?!
             self.block_template = None
+            self.block_template_error += 1
```

Comment:
I know it's not from this PR but I was gonna ask to log something here because we are capturing all exceptions, so we could have the log of it. But given that we still don't have logging here, maybe just create an issue to do this in the future?

Reply:
Just added it. Thanks!